ui/components/src/TimeSeriesTooltip/TooltipHeader.tsx

```diff
@@ -65,6 +65,8 @@ export const TooltipHeader = memo(function TooltipHeader({
   // TODO: accurately calc whether more series are outside scrollable region using yBuffer, avg series name length, TOOLTIP_MAX_HEIGHT
   const showAllSeriesToggle = totalSeries > 5;
 
+  const pinTooltipHelpText = isTooltipPinned ? 'Click to Unpin' : 'Click chart to Pin';
```

Comment:
`Click chart to pin`  and `Click chart to unpin`?  The font case seems weird to me